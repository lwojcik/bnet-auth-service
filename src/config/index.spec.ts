import * as config from './index';

describe('config', () => {
  it('should be defined', () => {
    expect(config).toMatchSnapshot();
  });
});
