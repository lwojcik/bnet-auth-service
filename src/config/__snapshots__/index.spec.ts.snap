// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config should be defined 1`] = `
Object {
  "appConfig": [Function],
  "authConfig": [Function],
  "battleNetConfig": [Function],
  "endpointsConfig": [Function],
  "redisConfig": [Function],
  "throttleConfig": [Function],
}
`;
